test/k8sT/Services.go

```diff
@@ -568,9 +568,7 @@ var _ = SkipDescribeIf(helpers.RunsOn54Kernel, "K8sServicesTest", func() {
 		})
 	})
 
-	SkipContextIf(func() bool {
-		return helpers.SkipQuarantined() && helpers.RunsOnNetNextKernel()
-	}, "Checks service across nodes", func() {
+	SkipContextIf(func() bool { return helpers.RunsOnNetNextKernel() }, "Checks service across nodes", func() {
```

Comment:
You need to remove all conditions for skipping.

Reply:
confused, are these tests meant to run against net-next or not?

Reply:
(meanwhile will remove condition as you suggested to trigger tests)

Reply:
Yes, they are meant to be run on net-next. `SkipContextIf(func() bool { return helpers.RunsOnNetNextKernel() }` reads as "skip context if it runs on net-next kernel".

Reply:
Yea, curious then why was that condition checked in in the first place? I mean if we are to quarantine the tests temporarily, then `SkipQuarantined` should do it? That's why I only removed that in the first place, I thought the other condition is how it's meant to run lol